src/templates/ja/2019/methodology.html

```diff
@@ -0,0 +1,433 @@
+
+      <p>
+        各テストの結果は、<a href="https://en.wikipedia.org/wiki/HAR_(file_format)">HARファイル</a>として公開され、ウェブページに関するメタデータを含むJSON形式のアーカイブファイルです。
+      </p>
+    </section>
+
+    <section>
+      <h3 id="lighthouse"><a href="#lighthouse" class="anchor-link">Lighthouse</a></h3>
+
+      <p>
+        <a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse</a>は、Googleが構築した自動化されたウェブサイト品質保証ツールです。最適化されていない画像やアクセスできないコンテンツなどのユーザー体験のアンチパターンが含まれていないことを確認するためにウェブページを監査します。
+      </p>
+
+      <p>
+        HTTP Archiveは、すべてのモバイルWebページでLighthouseの最新バージョンを実行しています - リソースが限られているため、デスクトップページは含まれていません。2019年7月のクロール時点で、HTTP Archiveは<a href="https://github.com/GoogleChrome/lighthouse/releases/tag/v5.1.0">5.1.0</a>バージョンのLighthouseを使用していました。
+      </p>
+
+      <p>
+        Lighthouseは<a href="#webpagetest">WebPageTest</a>の中から独自のテストとして実行されますが、独自の設定プロファイルを持っています。
+      </p>
+      <div class="table-wrap"><div class="table-wrap-container">
+      <table>
+        <thead>
+          <tr>
+            <th>設定</th>
+            <th>値</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>CPU slowdown</td>
+            <td>1x*</td>
+          </tr>
+          <tr>
+            <td>Download throughput</td>
+            <td>1.475 Mbps</td>
+          </tr>
+          <tr>
+            <td>Upload throughput</td>
+            <td>0.675 Mbps</td>
+          </tr>
+          <tr>
+            <td>RTT</td>
+            <td>150 ms</td>
+          </tr>
+        </tbody>
+      </table>
+      </div></div>
+
+      <p class="note">
+        * 注 Lighthouseは通常、CPUが4倍遅くなるように設定されていますが、WebPageTestの<a href="https://github.com/GoogleChrome/lighthouse/issues/9668#issuecomment-535134232">バグ</a>のため、テスト時には1倍になっていました。
+      </p>
+
+      <p>
+        LighthouseとHTTP Archiveで利用可能な監査の詳細については、<a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse開発者向けドキュメント</a>を参照してください。
+      </p>
+    </section>
+
+    <section>
+      <h3 id="wappalyzer"><a href="#wappalyzer" class="anchor-link">Wappalyzer</a></h3>
+
+      <p>
+        <a href="https://www.wappalyzer.com/">Wappalyzer</a>は、ウェブページで使用されている技術を検出するためのツールです。<a href="https://www.wappalyzer.com/technologies">65のカテゴリ</a>があり、JavaScriptフレームワークからCMSプラットフォーム、さらには暗号通貨の採掘者に至るまで、テストされた技術の範囲があります。1,200以上の技術がサポートされています。
+      </p>
+
+      <p>
+        HTTP Archiveは、すべてのウェブページに対して最新バージョンのWappalyzerを実行します。2019年7月現在、Web AlmanacはWappalyzerの<a href="https://github.com/AliasIO/Wappalyzer/releases/tag/v5.8.3">5.8.3バージョン</a>を使用しています。
+      </p>
+
+      <p>
+        Wappalyzerは、WordPress、Bootstrap、jQueryのような開発者ツールの人気を分析する多くの章を強力にしています。例えば、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='ecommerce')}}">Eコマース</a> と <a href="{{ url_for('chapter', year=year, lang=lang, chapter='cms')}}">CMS</a> の各章は、それぞれの <a href="https://www. wappalyzer.com/categories/ecommerce">Eコマース</a> と <a href="https://www.wappalyzer.com/categories/cms">CMS</a> の各チャプターは、Wappalyzer が検出した技術のカテゴリに大きく依存しています。
```

Comment:
Extra space breaking the link:
```suggestion
        Wappalyzerは、WordPress、Bootstrap、jQueryのような開発者ツールの人気を分析する多くの章を強力にしています。例えば、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='ecommerce')}}">Eコマース</a> と <a href="{{ url_for('chapter', year=year, lang=lang, chapter='cms')}}">CMS</a> の各章は、それぞれの <a href="https://www.wappalyzer.com/categories/ecommerce">Eコマース</a> と <a href="https://www.wappalyzer.com/categories/cms">CMS</a> の各チャプターは、Wappalyzer が検出した技術のカテゴリに大きく依存しています。
```

Reply:
Sorry, I missed.
I fixed.